packages/frontend/src/components/accounts/SetPassword/ui/Input/ui.ts

```diff
@@ -0,0 +1,46 @@
+import styled, {css} from 'styled-components';
+
+export const Wrapper = styled.div`
+    position: relative;
+`;
+
+export const InputContent = styled.div`
+`;
+
+export const InputElement = styled.input<{error: boolean}>`
+    margin-top: 0 !important;
+    
+    ${props => props.error && css`
+        box-shadow: 0px 0px 0px 2px rgba(242, 0, 0, 0.2);
+        border: 1px solid #DC3D43;
+        background: #ffffff;
+        
+        &:focus {
+            box-shadow: 0px 0px 0px 2px rgba(242, 0, 0, 0.2);
+            border: 1px solid #DC3D43;
+        }
+    `}
+`;
+
+export const IconClickArea = styled.div`
```

Comment:
We will remember about addition of some background under the icon

![bg](https://user-images.githubusercontent.com/61930014/198286288-8dee9aa7-6224-48c2-9c78-9d98a4b61f01.png)

Reply:
Fixed with padding-right on input
<img width="606" alt="image" src="https://user-images.githubusercontent.com/4532214/198344434-0080328b-37bb-4697-8a33-3ff6664b9bf6.png">